zkg

```diff
@@ -629,6 +630,25 @@ def cmd_install(manager, args, config, configfile):
             else:
                 print('Loaded "{}"'.format(name))
 
+    if args.runtime_dep_check:
```

Comment:
But args.nodeps is not a top-level flag. Do you want me to pull it to the top?

Reply:
Looking at the other comments, I guess we need to pull it at the top-level.

Reply:
Thought we already had `zkg install --nodeps`, so was thinking that we add `zkg load --nodeps` and `zkg unload --nodeps`.  Don't think it's suitable for a top-level `zkg --nodeps <cmd>` flag since there's some commands where it will never be relevant.

Reply:
Okay. I will fix it accordingly.